spec/api_spec.rb

```diff
@@ -1,20 +1,27 @@
 require 'spec_helper'
 require 'orchestrate'
+require 'timecop'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.